code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -83,6 +83,8 @@
 	name = "ammunition packet (9mm laser)"
 	icon_state = "pistol_si"
 	ammo_type = /obj/item/ammo_casing/pistol_35/laser
+	matter = list(MATERIAL_PLASTIC = 1)
```

Comment:
this mean that this type of box dosnt need cardboard

Reply:
Fully intended and just a QoL feature. Cardboard is annoying to get and it would make sense for Soteria to be a little bit more special

Reply:
Less common logic but more so common habit but fine. It is just an extra 5 to 10 minutes of chores added onto science that you wish to have and keep. _The science chore list already pretty fucking long._